noir-projects/aztec-nr/aztec/src/messages/processing/log_retrieval_response.nr

```diff
@@ -1,10 +1,9 @@
 use protocol_types::{
-    constants::{MAX_NOTE_HASHES_PER_TX, PRIVATE_LOG_CIPHERTEXT_LEN, PUBLIC_LOG_PLAINTEXT_LEN},
+    constants::{MAX_NOTE_HASHES_PER_TX, MESSAGE_CIPHERTEXT_LEN, PUBLIC_LOG_PLAINTEXT_LEN},
     traits::Deserialize,
 };
 
-global MAX_LOG_CONTENT_LEN: u32 =
-    std::cmp::max(PUBLIC_LOG_PLAINTEXT_LEN, PRIVATE_LOG_CIPHERTEXT_LEN);
+global MAX_LOG_CONTENT_LEN: u32 = std::cmp::max(PUBLIC_LOG_PLAINTEXT_LEN, MESSAGE_CIPHERTEXT_LEN);
```

Comment:
See how this reads a bit crazy - why are we measuring public log plaintext with 'message ciphertext'? We should be comparing pub and priv logs, no these. Also, is the content encrypted or not? Because if not it shold be the private _plaintext_.